rules/windows/defense_evasion_execution_suspicious_explorer_winword.toml

```diff
@@ -24,12 +24,14 @@ type = "query"
 
 query = '''
 event.category:process and event.type:(start or process_started) and
- process.pe.original_file_name:(WinWord.exe or EXPLORER.EXE or w3wp.exe or DISM.EXE) and
- not (process.name:(winword.exe or WINWORD.EXE or explorer.exe or w3wp.exe or Dism.exe) or
-             process.executable:("C:\Windows\explorer.exe" or C\:\\Program?Files\\Microsoft?Office\\root\\Office*\\WINWORD.EXE or 
-                C\:\\Program?Files?\(x86\)\\Microsoft?Office\\root\\Office*\\WINWORD.EXE or 
-                 "C:\Windows\System32\Dism.exe" or "C:\Windows\SysWOW64\Dism.exe" or 
-                  "C:\Windows\System32\inetsrv\w3wp.exe"))
+  (process.pe.original_file_name:(WinWord.exe or EXPLORER.EXE or w3wp.exe or DISM.EXE) or
+     winlog.event_data.OriginalFileName:(WinWord.exe or EXPLORER.EXE or w3wp.exe or DISM.EXE)) and
```

Comment:
I don't think it will ever evaluate true if they aren't adjusted.

Reply:
one nice thing about `original_file_name`, it will always match the exact case.

so we should check what the exact case is within the PE header, and use that here. if the case changes for the file, authenticode breaks. so we can assume that there is only 1 correct case. we just gotta get the right one.